matt-brian/server.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+// DONE: Initialize your project using NPM to create and populate a package.json file
+
+// DONE: Require the Express package that you installed via NPM, and instantiate the app
+// Remember to install express, and be sure that it's been added to your package.json as a dependency
+// There is also a package here called body-parser, which is required in for use in a new route.
+// Be sure to install that and save it as a dependency after you create your package.json.
+const EXPRESS = require('express');
+const bodyParser = require('body-parser').urlencoded({extended: true});
+const PORT = process.env.PORT || 3000;
+const app = EXPRESS();
+// DONE: Include all of the static resources as an argument to app.use()
+app.use(EXPRESS.static('public'));
+
+// TODO: (STRETCH) Write a new route that will handle a request and send the new.html file back to the user
+app.get('/', function(request, response){
```

Comment:
Since this path is the root of the directory, you cannot actually reach it. You need to create a different path in order to be able to reach the new.html file.